lib/middleman-cdn/commands.rb

```diff
@@ -72,7 +72,7 @@ def cdn_invalidate(*args)
 
       def self.say_status(cdn, status, newline: true, header: true, wait_enter: false)
         message = ""
-        message << "#{:cdn.to_s.rjust(12).green}  #{cdn.try(:yellow)}" if header
+        message << "#{ANSI.green { :cdn.to_s.rjust(12)} }  #{ANSI.yellow{ cdn } unless cdn.nil? }" if header
```

Comment:
There's unevent whitespace on the closing `}`s of the string interpolations. I can fix that post merge though, no big deal.